CMakeLists.txt

```diff
@@ -59,6 +59,12 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le|powerpc64le")
     if(${OQS_DIST_BUILD})
         set(OQS_DIST_PPC64LE_BUILD ON)
     endif()
+elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "(ppc64|powerpc64)")
+    set(ARCH "ppc64")
```

Comment:
What about adding a warning message here about "suboptimal" CI for this platform?

Reply:
Done.